schema.graphql

```diff
@@ -485,16 +585,94 @@ type MatchQuestionBuzz {
 """
 Docs group: Match
 """
-type MatchQuestionBonus {
+type BuzzPosition {
+  """
+  The time of the buzz. An ISO 8601-formatted string, with time zone, in the tournament's local time zone. May not be before the start time of reading the relevant question.
+  """
+  time: Date
+
+  """
+  The (`0`-based) index of the word on which the player buzzed. Hearing any part of a word should be considered the same as hearing the entire word. So if a question begins "Abraham Lincoln…" and the player buzzes on "Abra—", this value would be `0`.
+  """
+  word_index: Int
+
+  """
+  The (`0`-based) index of the clue on which the player buzzed. Clues may be defined in the `Question` object. Hearing any part of a clue should be considered the same as hearing the entire clue.
+  """
+  clue_index: Int
+}
+
+"""
+This object can be reused for every time the same question is read, i.e., it represents a question in general rather than a question being read in a specific match.
+
+Docs group: Question
+"""
+type Question {
   """
-  Which bonus number this was. May or may not match the `question_number` of the `MatchQuestion` parent, depending on whether bonuses are paired with tossups or simply read in order (and if the latter, on whether any bonus-earning questions failed to lead to a bonus).
+  The number of the question in the packet.
   """
   question_number: Int!
+  
+  """
+  The type of the question.
+  """
+  type: QuestionType!
+  
+  """
+  The number of parts the question has (typically `1` if `type` is `tossup` and `3` if `type` is `bonus`, but some formats may vary).
+  """
+  parts: Int
+  
+  """
+  The identifier for the question in some external system, if any (e.g., the question IDs that NAQT prints below each question in its packet).
+  """
+  external_id: String
+  
+  """
+  Freeform information about the question's category, e.g. `British History`.
```

Comment:
Besides category, other commonly tracked metadata for questions includes the canonical answer (or answers for bonuses), the authors or editors, and for tossups, the number of words and number of words in power.